setup.py

```diff
@@ -32,7 +32,8 @@
         'Click', 'pandas', 'PyYAML', 'python-dateutil',
         'pika==0.13.1', 'enum34', 'glob2', 'tqdm', 'lxml',
         'tblib', 'docker', 'requests', 'seaborn', 'paramiko',
-        'matplotlib', 'kubernetes', 'ibm-cos-sdk', 'gcsfs'
+        'matplotlib', 'kubernetes', 'ibm-cos-sdk', 'gcsfs',
+        'aliyun-fc2', 'oss2',
```

Comment:
@JosepSampe i wonder about the following. Running setup will install all dependencies, like gcfs, aliyun-fc2 and in the future all other backends. Is there way to install only what is needed? For example when i use Lithops i need ibm-cos-sdk, but i don't need gcfs or aliyun or anything else. The same i guess if someone using gcfs he doesn't need ibm-cos-sdk, etc. Is there way we install packages based on the backend we intend to use? May be we can have different groups of dependencies and choosing some parameter will install them. Otherwise i start to like it less that every time we run setup, ALL the dependencies being installed

Reply:
@gilv Yes, we can create groups of installation. I will add them later

Reply:
@gilv Yep ;)